app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -630,40 +630,72 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
+                                            deleteMessage(getSelectedItems().get(i));
+                                        }
+                                        Toast toast = Toast.makeText(recyclerView.getContext() , "Message deleted", Toast.LENGTH_LONG);
+                                        toast.setGravity(Gravity.CENTER, 0, 0);
+                                        toast.show();
+                                        actionMode.finish();
                                     }
-                                    if (getSelectedItems().size() == 1)
-                                        Snackbar.make(recyclerView, " Message Deleted !!", Snackbar.LENGTH_LONG).show();
-                                    else
-                                        Snackbar.make(recyclerView, getSelectedItems().size() + " Messages Deleted !!", Snackbar.LENGTH_LONG).show();
-
-                                    actionMode.finish();
-
-                                }
-                            }).
-                            setNegativeButton("Cancel", new DialogInterface.OnClickListener() {
-                                @Override
-                                public void onClick(DialogInterface dialog, int which) {
-                                    dialog.cancel();
-                                }
-                            });
-
-                    AlertDialog alert = d.create();
-                    alert.setTitle("Delete");
-                    alert.show();
-                    Button cancel = alert.getButton(DialogInterface.BUTTON_NEGATIVE);
-                    cancel.setTextColor(Color.BLUE);
-                    Button delete = alert.getButton(DialogInterface.BUTTON_POSITIVE);
-                    delete.setTextColor(Color.RED);
-                    return true;
+                                }).
+                                setNegativeButton("Cancel", new DialogInterface.OnClickListener() {
+                                    @Override
+                                    public void onClick(DialogInterface dialog, int which) {
+                                        dialog.cancel();
+                                    }
+                                });
+
+                        AlertDialog alert = d.create();
+                        alert.show();
+                        Button cancel = alert.getButton(DialogInterface.BUTTON_NEGATIVE);
+                        cancel.setTextColor(Color.BLUE);
+                        Button delete = alert.getButton(DialogInterface.BUTTON_POSITIVE);
+                        delete.setTextColor(Color.RED);
+                        return true;
+
+                    }
+                    else {
+                        AlertDialog.Builder d = new AlertDialog.Builder(context);
+                        d.setMessage("Delete "+ getSelectedItems().size() + " messages?").
+                                setCancelable(false).
```

Comment:
@saketkumar95 I think there is no need of writing all code of alert dialog box in else statement. It can be written once and you can just write 
```
if (getSelectedItems().size() == 1){  
    d.setMessage("Delete message?")
} else { 
    d.setMessage("Delete "+ getSelectedItems().size() + " messages?") 
}
```
same while showing toast.